background.ts

```diff
@@ -894,6 +895,11 @@ class BackgroundCommandWorker implements CommandWorkerInterface {
       return ['no changes necessary', ''];
     }
 
+    const allowListConf = BackendHelper.createImageAllowListConf(cfg.containerEngine.imageAllowList);
+
+    await k8smanager.executor.writeFile(`/usr/local/openresty/nginx/conf/image-allow-list.conf`, allowListConf, 0o644);
+    await k8smanager.executor.execCommand({ root: true }, '/sbin/rc-service', 'openresty', 'reload');
+
```

Comment:
It has nothing to do with Kubernetes. It should go into a `Backend` base class, but there doesn't seem to be such a thing. Just an interface definition, but no implementation inheritance.

What exactly is the place it should go? And I don't want to duplicate the code between backends; I think the common code should be factored out and shared.

Reply:
Consider having a new backend helper class, then? It's inappropriate in `background.ts` mainly because it's already including too many things (it's the main entry point of the app, after all). We should be trying to move things away from here, rather than adding more.